### ISSUE TITLE
Narrator is not announcing the role of the repository and license link.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/vscode#105726]
["Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11y_VSCode;#WCAG1.3.1; #A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;

Environment Details:​
Application Name: VS Codespace
Operating System: Win10​
Version: 1.47.0-insider (user setup)
Commit: d5e9aa0227e057a60c82568bf31c04730dc15dcd
Date: 2020-07-08T10:52:52.439Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

Additional Details:
MAS References:WCAG1.3.1

NVDA Version: 2020.1

**Repro Steps:​**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Codespace"
3. Click on "Visual Studio codespace".
4.Launch Narrator.
5.Navigate up to repository and license links and observed that narrator is announcing the role or not.

**Actual:**
Narrator is announcing repository link as only repository and same for license.

**Same issue has been observed for Visual Studio Codespace heading link.** 

**Expected: ​**
Narrator should announce like repository link and license link.

**​User Impact: ​**
Screen reader only users will face difficulty to understand the controls properly.

**Attachment link for Reference:**

**Comment by crazydaisy925** [comment]
I'm Becka Mullins

**Comment by isidorn** [comment]
Also we do not want testing to be done on narrator.

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!
